istioctl/cmd/auth_test.go

```diff
@@ -17,13 +17,28 @@ package cmd
 import (
 	"bytes"
 	"fmt"
+	"reflect"
 	"strings"
 	"testing"
 
+	"istio.io/istio/istioctl/pkg/auth"
+
 	"istio.io/istio/pilot/test/util"
 )
 
-func runCommandAndCheckGolden(name, command, golden string, t *testing.T) {
+func runCommandAndCheckGoldenFile(name, command, golden string, t *testing.T) {
+	out := runCommand(name, command, t)
+	util.CompareContent(out.Bytes(), golden, t)
+}
+
+func runCommandAndCheckExpectedString(name, command, expected string, t *testing.T) {
+	out := runCommand(name, command, t)
+	if !reflect.DeepEqual(out.String(), expected) {
```

Comment:
Why are you using DeepEqual() to compare strings?

Reply:
Will change to use ==